### PR TITLE
Update to 2.3.0

### PR DESCRIPTION
**Pull request by jgiannuzzi**
Storm 2.3.0 has been released: http://storm.apache.org/2021/09/27/storm230-released.html

### CODE REVIEW

File: 2.3.0/Dockerfile [hunk 1]
```diff
@@ -28,8 +28,8 @@ RUN set -eux; \
 # Verify that gosu binary works
     gosu nobody true
 
-ARG GPG_KEY=79B03D059E628478FC9F1D8B152CAD0C46E87B61
-ARG DISTRO_NAME=apache-storm-2.2.0
+ARG GPG_KEY=5167DE337E7370373499FC1DA4A672F11B5050C8
```
Comment (jgiannuzzi) [inline 2.3.0/Dockerfile:31]: This key is one of the official signing keys: https://downloads.apache.org/storm/KEYS

File: 2.3.0/Dockerfile [hunk 2]
```diff
@@ -54,6 +54,7 @@ RUN set -eux; \
     ddist "$DISTRO_NAME.tar.gz" "storm/$DISTRO_NAME/$DISTRO_NAME.tar.gz"; \
     ddist "$DISTRO_NAME.tar.gz.asc" "storm/$DISTRO_NAME/$DISTRO_NAME.tar.gz.asc"; \
     export GNUPGHOME="$(mktemp -d)"; \
+    gpg --keyserver keyserver.ubuntu.com --recv-key "$GPG_KEY" || \
```
Comment (jgiannuzzi) [inline 2.3.0/Dockerfile:57]: This keyserver seems more available than the other ones these days

**Comment by jgiannuzzi** [comment]
Hi @31z4, could you please review this PR?

I'm happy to create a PR on https://github.com/docker-library/official-images once merged if it helps 😄

**Comment by mariannovotny** [comment]
Hi @31z4 Could you please look into this PR?
We are going to upgrade Storm to 2.3.0 as soon as possible because of improvements and vulnerabilities in Storm 2.2.0 :
- CVE-2021-40865: Apache Storm: Unsafe Pre-Authentication Deserialization In Workers (critical severity,  https://nvd.nist.gov/vuln/detail/CVE-2021-40865)
- CVE-2021-38294: Apache Storm: Shell Command Injection Vulnerability in Nimbus Thrift Server(critical severity https://nvd.nist.gov/vuln/detail/CVE-2021-38294)

**Comment by 31z4** [comment]
Thanks a lot for your contribution @jgiannuzzi 👍 And sorry for the delayed response.

**Comment by jgiannuzzi** [comment]
Thanks for the merge @31z4!

Would you like me to create the PR for https://github.com/docker-library/official-images or will you take care of it?

**Comment by 31z4** [comment]
I just created it: https://github.com/docker-library/official-images/pull/11235

**Pull request merged**